### PR TITLE
fix(test): added support for prod-preview

### PR DESCRIPTION
**Pull request by rgarg1**
The Planner API tests can now run against either production or prod-preview.

### CODE REVIEW

File: EE_API_automation/pytest/README.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+A sample Python project
```
Comment (jarifibrahim) [inline EE_API_automation/pytest/README.rst]: We should remove this file. This is a sample file.

File: EE_API_automation/pytest/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2016 The Python Packaging Authority (PyPA)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of
+this software and associated documentation files (the "Software"), to deal in
+the Software without restriction, including without limitation the rights to
+use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
```
Comment (jarifibrahim) [inline EE_API_automation/pytest/LICENSE.txt]: I'm not sure if we should add this License file.

**Comment by jarifibrahim** [comment]
The PR works as expected, we just have to do a few cosmetic changes :+1:

**Comment by rgarg1** [comment]
@jarifibrahim Changed!!

**Pull request merged**